python/sdss_access/path/path.py

```diff
@@ -1125,13 +1125,66 @@ def healpixgrp(self, filetype, **kwargs):
         Returns
         -------
         healpixgrp : str
-            HEALPix group directory, HEALPix//100.
+            HEALPix group directory, HEALPix//1000.
 
         '''
 
         healpix = int(kwargs['healpix'])
-        subdir = "{:d}".format(healpix//100)
+        subdir = "{:d}".format(healpix//1000)
         return subdir
 
+    def apgprefix(self, filetype, **kwargs):
+        ''' Returns APOGEE prefix using telescope/instrument.
+
+        Parameters
+        ----------
+        filetype : str
+            File type parameter.
+        telescope : str
+            The APOGEE telescope (apo25m, lco25m, apo1m).
+        instrument : str
+            The APOGEE instrument (apogee-n, apogee-s).
+
+        Returns
+        -------
+        prefix : str
+            The APOGEE prefix (ap/as).
+
+        '''
+
+        telescope = kwargs.get('telescope')
+        if telescope is not None:
+            prefix = {'apo25m':'ap', 'apo1m':'ap', 'lco25m':'as'}[telescope]
+            return prefix
+        instrument = kwargs.get('instrument')
+        if instrument is not None:
+            prefix = {'apogee-n':'ap', 'apogee-s':'as'}[instrument]
+            return prefix
+        return ''
+
+    def apginst(self, filetype, **kwargs):
+        ''' Returns APOGEE "instrument" from "telescope".
```

Comment:
For this function to work, it needs a `telescope` kwarg.  Are there any paths that do not have `telescope` but have `instrument` as a kwarg?  If so, is the plan to migrate those over to use `telescope` and this function?